src/test/lib/EventTest.cpp

```diff
@@ -17,6 +17,9 @@
 #define QUIC_EVENT_ACTION_SHUTDOWN_CONNECTION   1
 #define QUIC_EVENT_ACTION_SHUTDOWN_STREAM       2
 
+const uint8_t StreamPayload[64] = {0x1};
```

Comment:
This will only set the first byte to `0x1`, it won't initialize the entire array to `0x1`. Just checking assumptions.

Reply:
I don't care really. :) I just put something here.